weave

```diff
@@ -1747,6 +1747,7 @@ launch_plugin_if_not_running() {
     if ! PLUGIN_CONTAINER=$(docker run -d --name=$PLUGIN_CONTAINER_NAME \
         $(docker_run_options) \
         $RESTART_POLICY \
+        --pid=host \
```

Comment:
Short comment either here or in the commit message would be useful for why we start container in host PID namespace.